spec/simplecov_spec.rb

```diff
@@ -229,4 +229,27 @@
       end
     end
   end
+
+  # Normally wouldn't test private methods but just start has side effects that
+  # cause errors so for time this is pragmatic (tm)
+  describe ".start_coverage_measurement", :if => SimpleCov.coverage_start_arguments_supported? do
+    before :each do
+      # global state, yay!
```

Comment:
cheeky comment, but it doesn't really explain why `@coverage_criteria` keeps being stored and needs to be cleared

Reply:
ah, I'll add some more then. It's stored in the eigenclass object of SimpleCov itself and that isn't restored between test runs. It's basically a Singleton object global.

IPart of me would like to change this, the other part of me knows that it might break too many things.